src/GrumPHP/Collection/FilesCollection.php

```diff
@@ -73,6 +73,22 @@ public function path($pattern)
     }
 
     /**
+     * Filter by paths
+     *
+     * $collection->paths(array('/^spec\/','/^src\/'))
+     *
+     * @param array $patterns
+     *
+     * @return FilesCollection
+     */
+    public function paths(array $patterns)
```

Comment:
This code is almost the same as the regular `path()` method. Can you let the `path()` method use the `paths()` method instead of duplicating the code?

There should also be a spec test for this new feature.
Note that we are moving away from the old `array()` syntax and are using the new short array syntax instead. Can you replace that one?